Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Circular.ProgressBar/Globals/GlobalDeclarations.cs

```diff
@@ -0,0 +1,18 @@
+﻿#region BSD License
```

Comment:
The license needs to remain in all code files

Reply:
Done